transit_odp/data_quality/pti/functions.py

```diff
@@ -270,22 +286,15 @@ def has_flexible_or_standard_service(context, services):
 
 def has_flexible_service_classification(context, services):
     """
-    Check when file has detected a flexible service (includes a
-    FlexibleJourneyPattern or includes a BookingArrangements element),
-    it has ServiceClassification and Flexible elements.
+    Check when file has detected a flexible service (includes
+    FlexibleService), it has ServiceClassification and Flexible elements.
     If the file also has a standard service, then return True.
     """
     for service in services:
         ns = {"x": service.nsmap.get(None)}
-        flexible_journey_pattern_list = service.xpath(
-            "x:FlexibleService/x:FlexibleJourneyPattern", namespaces=ns
-        )
-        booking_arrangements_list = service.xpath(
-            "x:FlexibleService/x:FlexibleJourneyPattern/x:BookingArrangements",
-            namespaces=ns,
-        )
+        flexible_service_list = service.xpath("x:FlexibleService", namespaces=ns)
```

Comment:
this can be resolved - if a service does not have a ServiceClassification element then it should be highlighted in the validation rule which has been added